spec/testing/context_factory_spec.rb

```diff
@@ -70,7 +70,7 @@
       LightService::Testing::ContextFactory
       .make_from(organizer)
       .for(TestDoubles::AddsTwoAction)
-      .with(number: 2)
```

Comment:
Same here, I'd just "please" Rubocop to allow the rule violation on next line only.